packages/pyolite-kernel/package.json

```diff
@@ -64,9 +64,12 @@
     "access": "public"
   },
   "pyolite": {
-    "packages": [
-      "py/pyolite"
-    ]
+    "packages": {
+      "py/pyolite": "0.1.0",
+      "py/ipykernel": "5.5.5",
+      "py/nbformat": "4.2.0",
+      "py/widgetsnbextension": "3.5.0"
```

Comment:
Maybe as a follow-up we could find a way to keep the versions in sync with the `__version__` defined in the packages.

Reply:
Yeah, the #147 piece will grow the ability mess about with `.whl` files... we can figure out some way to sync the versions (maybe a _dumb_ `requirements.txt`, (no `-r` or `git` shenanigans).